dxr/static/css/dxr.css

```diff
@@ -332,7 +332,7 @@ mark {
     transform: rotate3d(0, 0, 1, 270deg);
 }
 .content {
-    margin-top: 55px;
+    margin: 55px 0 25px;
```

Comment:
Did you really mean to change the right and left margins to 0? That's what happened here. (See http://www.htmldog.com/reference/cssproperties/margin/ and the meaning of 3-value margin arguments.)